MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -81,26 +81,29 @@ class QuestionFactory: QuestionFactoryProtocol {
             guard let movie = self.movies[safe: index] else { return }
             
             var imageData = Data()
-           
-           do {
+            
+            do {
                 imageData = try Data(contentsOf: movie.resizedImageURL)
             } catch {
                 print("Failed to load image")
             }
             
             let rating = Float(movie.rating) ?? 0
             
-            let ratingTexts: [String] = ["Рейтинг этого фильма больше чем 8?","Рейтинг этого фильма больше чем 8.5?","Рейтинг этого фильма больше чем 9?","Рейтинг этого фильма больше чем 9.5?","Рейтинг этого фильма больше чем 7?","Рейтинг этого фильма больше чем 7.5?"]
             
-            let text = ratingTexts.randomElement()
+            let randomRate = (Float.random(in: 8.0...9.2)*10).rounded()/10
```

Comment:
Принято оставлять пробелы вокруг операторов 
```suggestion
            let randomRate = (Float.random(in: 8.0...9.2) * 10).rounded() / 10
```